calculator/operations.py

```diff
@@ -1,18 +1,15 @@
+from functools import reduce
 
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x,y: x-y, args[1:], args[0])
 
 def multiply(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x,y: x*y, args, 1)
 
 def divide(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x,y: float(x)/y, args[1:], args[0])
 
```

Comment:
If the first argument of your reduce iterable is the starting point of your reduce function you don't need the last argument. 

    return reduce(lambda x,y: x-y, args)
    return reduce(lambda x,y:x*y, args)
    return reduce(lambda x,y:float(x)/y, args)

Just FYI. Yours works fine, I just thought I'd drop a note.